distros/ubuntu1604/zoneminder.postinst

```diff
@@ -30,25 +30,27 @@ if [ "$1" = "configure" ]; then
         #
         # Get mysql started if it isn't running
         #
-        $(systemctl status mysql.service >/dev/null 2>&1); MYSQL_STATUS=$?
-        $(systemctl status mariadb.service >/dev/null 2>&1); MARIADB_STATUS=$?
         
-        # 3 = inactive, 4 = nonexistant
-        if [ "$MARIADB_STATUS" != "4" ]; then
+        if [ -e "/lib/systemd/system/mariadb.service" ]; then
```

Comment:
This looks to see if mariadb is installed, but it could be disabled.  Perhaps we should check instead in /etc/systemd/system/multi-user.target.wants/mariadb.service